### ISSUE TITLE
Unable to replicate default training on MNIST

### ISSUE THREAD

**Issue by Sun-Jiatuhao** [RorschachChen/entangled-watermark-torch#1]
hello，I tried 
```python
python train-origin.py --dataset mnist 
``` 
but results are 
<details><summary>Details</summary>
<p>

Extracted Model || validation accuracy: 97.18, watermark success: 0.025472005208333332

</p>
</details> 
It is far from the result in the paper
the logs are：
```python
Namespace(batch_size=512, config='configs/mnist.json', ratio=1, lr=0.001, lr_drop=100, weight_decay=1e-05, epochs=10, w_epochs=10, dataset='mnist', model_type='2_conv', metric='cosine', factors=[32, 32, 32], temperatures=[1, 1, 1], threshold=0.1, maxiter=10, w_lr=0.01, t_lr=0.1, source=1, target=7, shuffle=0, seed=0, verbose=1, default=1, layers=18, distribution='out', input_channel=1, output_channel=10, device='cuda', output_dir='logs', resume=False, start_epoch=0, num_workers=2)
Loading dataset: mnist
ewe_model normal train finished in 31.823971033096313s. 
optimize trigger finished in 20.804205417633057s. 
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████| 49/49 [14:28<00:00, 17.73s/it]
watermark finished in 868.9898121356964s. 
Victim Model || validation accuracy: 98.03, watermark success: 0.9998372395833334
train extract model finished in 67.61916613578796s. 
Extracted Model || validation accuracy: 97.18, watermark success: 0.025472005208333332
train clean model finished in 66.46442699432373
Clean Model || validation accuracy: 97.25, watermark success: 0.021321614583333332
```
